weave/ops_arrow/list_.py

```diff
@@ -760,6 +578,20 @@ def _map_column(
             mapped = with_mapped_children
         return mapped
 
+    def _decode_gql_types(self) -> "ArrowWeaveList":
```

Comment:
This seems like a super expensive procedure in terms of space - we might want to warn/guard against this

Reply:
Yes its very bad. It should never be used. We should rewrite unnest to avoid doing it.